checks/system/unix.py

```diff
@@ -221,6 +221,31 @@ def check(self, agentConfig):
             return False
 
 
+class File_Handlers(Check):
+
+    def check(self, agentConfig):
+
+        if not Platform.is_linux():
+            return False
+
+        try:
+            with open('/proc/sys/fs/file-nr', 'r') as file_handlers:
+                handler_contents = file_handlers.read()
+        except Exception:
+            self.logger.exception("Cannot extract system file handler stats")
+            return False
+
+        handler_metrics = handler_contents.split()
+
+        # https://www.kernel.org/doc/Documentation/sysctl/fs.txt
+        allocated_fh = float(handler_metrics[0])
+        allocated_unused_fh = float(handler_metrics[1])
+        max_fh = float(handler_metrics[2])
+
+        fh_in_use = (allocated_fh - allocated_unused_fh) / max_fh
+
+        return {'system.file_handles_in_use': float(fh_in_use)}
```

Comment:
I _think_ we could name it `system.fs.file_handles.in_use` instead, what do you think? (we already have a few `system.fs.` metrics, I think it makes to put this new metric in that metric namespace too since it's collected from `/proc/sys/fs/` too)